lib/driver/base.js

```diff
@@ -257,8 +258,8 @@ module.exports = Base = Class.extend({
     }
   },
 
-  addMigrationRecord: function (name, callback) {
-    this.runSql('INSERT INTO "' + global.migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
+  addMigrationRecord: function (name, migrationTable, callback) {
+    this.runSql('INSERT INTO "' + migrationTable + '" (name, run_on) VALUES (?, ?)', [name, new Date()], callback);
```

Comment:
Do you want global.migrationTable everywhere that I use it instead of passing a param? Or just in the base.js?